### PR TITLE
Include workaround reg. switching to new device

### LINKED ISSUE #411: Add entry to FAQ: Recommended procedure when switching to a new phone
I will shortly be switching to a new phone so I was wondering what the correct procedure in terms of the CWA is. I expected a FAQ entry on this, but there is just a mention of an iOS bug if one uses iTunes to migrate the app with the recommendation to simply install the new one.

In the first place, there should definitely be a FAQ entry on this. As to the content of the FAQ entry:

- First, the new FAQ entry should mention that the data on collected and sent keys can currently not be transferred to a new phone on both Android and iOS, and that the CWA should simply be newly installed on the new phone as soon as possible in all cases.

- Secondly, one could give the advice to those that wanted to maximise continuity to carry both phones with them for a period of two weeks.  In case of a positive test result within those two weeks, the old phone should be used to scan the QR code and submit the diagnosis keys.

- Alternatively, one could advise that the old phone could be left at home and only be turned on once a day for two weeks to check for risk encounters. This would be sufficient to receive warnings about risk encounters from days where only the old phone was carried; however, in the unlikely event of receiving a positive test result oneself, one would require an extra TeleTAN to be able to submit diagnosis keys from both phones (I'm not sure whether this is supposed to be possible in such a case).

Note that there are some related issues already (without proper resolution), but those are not concrete requests for a FAQ entry, so I opened an issue her:

- https://github.com/corona-warn-app/cwa-wishlist/issues/37

- https://github.com/corona-warn-app/cwa-documentation/issues/82

### PR DESCRIPTION
**Pull request by Ein-Tim**
I included the text I proposed [here](https://github.com/corona-warn-app/cwa-website/issues/411#issuecomment-740747515) in the FAQ entry(s):

- https://www.coronawarn.app/de/faq/#backup
- https://www.coronawarn.app/en/faq/#backup

I hope I handled the Links okay, if not lemme now or feel free to change something (edit by maintainers is allowed)

Sorry for the typo in the commit 😀

Related: #411

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -432,7 +432,10 @@
                         "anchor": "backup",
                         "active": true,
                         "textblock": [
-                            "Nein, diese sind nicht Bestandteil der Datensicherung. Das heißt, die gesammelten Zufalls-IDs können auch nicht auf dem selben oder einem anderen Gerät wiederhergestellt werden."
+                            "Nein, diese sind nicht Bestandteil der Datensicherung. Das heißt, die gesammelten Zufalls-IDs können auch nicht auf dem selben oder einem anderen Gerät wiederhergestellt werden.",
+                            "Falls Sie zu einem neuem Handy gewechselt haben und über alle Kontakte informiert werden wollen ist es notwendig, dass sie auf ihrem alten Gerät die Corona-Warn App noch 14 Tage lang regelmäßig öffnen, um zu sehen, ob diese Ihnen Begegnungen anzeigt.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:436]: "dass sie auf ihrem alten Gerät" => "dass Sie auf Ihrem alten Gerät"

File: src/data/faq_de.json [hunk 2]
```diff
@@ -432,7 +432,10 @@
...
+                            "Sollten Sie innerhalb dieser 14 Tage positiv getestet werden, sollten Sie das Testergebnis auf beiden Geräten teilen, damit alle Kontakte informiert werden können.",
+                            "Eine Anleitung, wie sie Ihr Testergebnis auf mehreren Geräten teilen könnn, finden Sie <a href='#test_multiple_devices' target='_blank' rel='noopener'>Wie teile ich ein positives Testergebnis auf mehreren Geräten?</a>."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:438]: "wie sie Ihr Testergebnis auf mehreren Geräten teilen könnn" => "wie Sie Ihr Testergebnis auf mehreren Geräten teilen können"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
There are several typos in the German text to be corrected.
You can also correct the typo in the PR title "devide" => "device" which you should be able to edit independently from the original commit text.

Did you test using npm according to the https://github.com/corona-warn-app/cwa-website/blob/master/README.md instructions?

If you do that it is already really helpful to have a screen shot of the new web page pasted into the PR to review, since it is quite hard to look at the long lines of text in the .json files.

**Comment by Ein-Tim** [comment]
@MikeMcC399 
I will mark this PR as WIP since I'm a bit bussy at the moment... Will come back to you at the weekend (I will also share the screenshots at the weekend).

I also thought that I would maybe would be better to have a seperate FAQ Entry regarding this...
@dsarkar Could you give your opinion regarding this? 

Thanks!

**Comment by dsarkar** [comment]
@Ein-Tim, @MikeMcC399, we will have a look at this. Looks good so far. Will come back to you. DS

**Comment by dsarkar** [comment]
image
image

**Comment by Ein-Tim** [comment]
Mark to myself:
Include `all contacts` in the English text 😀

**Comment by Ein-Tim** [comment]
Since @heinezen already [started to draft](https://github.com/corona-warn-app/cwa-website/issues/411#issuecomment-742561389) an own FAQ Entry about this, I will close this PR.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Perhaps leave this in draft mode and then modify it later to refer to the main FAQ article about migrating to a new device? It would still have some value to refer to what to do in a backup article.

**Review by loki-cui — APPROVED** [review]

**Comment by Ein-Tim** [comment]
Since #653 was merged, the changes I made until now are obsolete. @MikeMcC399, I'm waiting for your proposal 😀
Thanks for the good cooperation!

**Comment by MikeMcC399** [comment]
@Ein-Tim 
The FAQ article https://www.coronawarn.app/en/faq/#new_device is directly below the article https://www.coronawarn.app/en/faq/#backup, so there is a high probability that a reader of the backup article will see the one about a new device as well. Also if a user searches for the term "backup" it will bring up both articles.

In that case I would suggest not making any changes to the backup article (like a cross-reference to the adjacent new device article). 

Readers have the relevant information available to them without any further changes.

Thanks once again for your active involvement. 👍 

image

**Comment by Ein-Tim** [comment]
Okay, thanks for the feedback. Closing this now 👍

**Pull request closed without merging**